files/ko/web/javascript/reference/global_objects/internalerror/index.html

```diff
@@ -1,121 +1,92 @@
- <dd>선택적 파라메터. 예외(Exception)이 발생한 코드의 라인 넘버.</dd>
+ <dt>{{jsxref("InternalError/InternalError", "InternalError()")}}</dt>
+ <dd>새로운 <code>InternalError</code> 객체를 만듭니다.</dd>
 </dl>
 
-<h2 id="설명">설명</h2>
-
-<p><code>InternalError</code> 는 JavaScript 엔진에서 내부 에러가 발생할 때마다 던져집니다.</p>
-
-<p>다음의 에러 예시 들의 경우는 일반적으로 어떤 값이 너무 큰 경우 입니다.</p>
-
-<ul>
- <li>"too many switch cases", (swich case의 수가 너무 많음.)</li>
- <li>"too many parentheses in regular expression", (정규표현식에 너무 많은 괄호가 있음.)</li>
- <li>"array initializer too large", (배열 초기화 값이 너무 큼.)</li>
- <li>"too much recursion". (너무 많은 재귀 호출.)</li>
-</ul>
-
-<h2 id="속성">속성</h2>
+<h2 id="Instance_properties">인스턴스 속성</h2>
 
 <dl>
- <dt>{{jsxref("InternalError.prototype")}}</dt>
- <dd><code>InternalError</code> 객체에 속성을 추가하도록 해줍니다.</dd>
+  <dt>{{jsxref("Error.prototype.message", "InternalError.prototype.message")}}</dt>
+  <dd>오류 메시지.{{jsxref("Error")}}로부터 상속되었습니다.</dd>
+  <dt>{{jsxref("Error.prototype.name", "InternalError.prototype.name")}}</dt>
+  <dd>오류 이름. {{jsxref("Error")}}로부터 상속되었습니다.</dd>
+  <dt>{{jsxref("Error.prototype.fileName", "InternalError.prototype.fileName")}}</dt>
+  <dd>오류가 발생한 파일 경로. {{jsxref("Error")}}로부터 상속되었습니다.</dd>
+  <dt>{{jsxref("Error.prototype.lineNumber", "InternalError.prototype.lineNumber")}}</dt>
+  <dd>오류가 발생한 곳의 줄 위치. {{jsxref("Error")}}로부터 상속되었습니다.</dd>
+  <dt>{{jsxref("Error.prototype.columnNumber", "InternalError.prototype.columnNumber")}}</dt>
+  <dd>오류가 발생한 곳의 열 위치. {{jsxref("Error")}}로부터 상속되었습니다.</dd>
+  <dt>{{jsxref("Error.prototype.stack", "InternalError.prototype.stack")}}</dt>
+  <dd>스택 추적. {{jsxref("Error")}}로부터 상속되었습니다.</dd>
 </dl>
 
-<h2 id="메소드">메소드</h2>
+<h2 id="Examples">예제</h2>
 
-<p>전역(global)의 <code>InternalError</code> 는 고유의 함수를 가지고 있지 않지만, 몇 가지의 메소드를 프로토타입 체인으로부터 상속받습니다.</p>
+<h3 id="Too_much_recursion">너무 많은 재귀 호출</h3>
 
-<h2 id="InternalError_인스턴스"><code>InternalError</code> 인스턴스</h2>
+<p>이 재귀 함수는 종료 조건에 따라 10번 수행됩니다.</p>
 
-<h3 id="속성_2">속성</h3>
+<pre class="brush: js">function loop(x) {
+  if (x &gt;= 10) // "x &gt;= 10" 는 종료 조건입니다.
+    return;
+  // 어떤 코드
+  loop(x + 1); // 재귀 호출
+}
+loop(0);</pre>
 
-<div>{{page('/en-US/docs/Web/JavaScript/Reference/Global_Objects/InternalError/prototype', 'Properties')}}</div>
+<p>이 조건을 매우 높은 값으로 설정하면 작동하지 않습니다:</p>
 
-<h3 id="메소드_2">메소드</h3>
+<pre class="brush: js example-bad">function loop(x) {
+  if (x &gt;= 1000000000000)
+    return;
+  // 어떤 코드
+  loop(x + 1);
+}
+loop(0);
 
-<div>{{page('/en-US/docs/Web/JavaScript/Reference/Global_Objects/InternalError/prototype', 'Methods')}}</div>
+// InternalError: too much recursion(너무 많은 재귀 호출)</pre>
 
-<h2 id="specifications">명세</h2>
+<p>더 많은 정보를 보려면 <a href="/ko/docs/Web/JavaScript/Reference/Errors/Too_much_recursion">InternalError: too much recursion(너무 많은 재귀 호출)</a>를 보시길 바랍니다.</p>
 
-<p>Not part of any specifications.</p>
+<h2 id="Specifications">명세</h2>
 
-<h2 id="browser_compatibility">브라우저 호환성</h2>
+<p>어떤 표준에도 속하지 않았습니다.</p>
```

Comment:
```suggestion
<p>어떤 표준에도 속하지 않습니다.</p>
```

과거형 표현은 현재 상태의 불확실성을 줄 수 있으므로 없애고 추후에 표준에 등록되면 그 때 반영하는게 좋다고 생각합니다.

아래 표현도 좋은 것 같아요 :)

> 표준의 일부가 아닙니다.

Reply:
저도 직역하면 표준의 일부가 아니라는 말이 되는데 저는 그게 더 이상하더라구요. 말씀하신대로 과거형보다는 현재형이 좋겠습니다.